chapters/inheritance.tex

```diff
@@ -1060,8 +1060,10 @@ \subsection{Annotation Choices for Suggested Redeclarations and Modifications}\l
 input the values \lstinline!false! or \lstinline!true! in the graphical user interface.
 
 Choices menus of replaceable elements can be automatically constructed
-showing the names of all classes that are either directly or indirectly
-derived by inheritance from the constraining class of the declaration.
+and at least is should be possible to show the names of all classes, \lstinline!A.B.$\ldots$.X.Z!, 
+that are either directly or indirectly
+derived by inheritance from the constraining class of the declaration, where \lstinline!A! to \lstinline!X!
+are non-partial packages, and \lstinline!Z! is non-partial.
 This can be recommended by having \lstinline!annotation choicesAllMatching = true!; and disabled by having
 \lstinline!annotation choicesAllMatching = false!.
```

Comment:
With the discussion fresh in mind, I get the intended meaning of _be possible to show_, but I'm not sure someone who wasn't part of the discussion would understand.

Maybe it's easier to write something where the annotation is mentioned first:
> Given the annotation `choicesAllMatching = true` on a replaceable element, tools are requested to construct a menu with choices of elements usable for replacement.  Exact criteria for inclusion in such a menu are not defined, but there shall be a a way to at least get a selection of classes `A.B.….X.Z` that are either…

Reply:
Yes, that makes a lot more sense. I will update accordingly.